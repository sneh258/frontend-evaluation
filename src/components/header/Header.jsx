import React from 'react';
import './header.css';

export default function Header() {
  return (
    <div className="header">
      <div className="header_name">EVENTIFY</div>
    </div>
  );
}
