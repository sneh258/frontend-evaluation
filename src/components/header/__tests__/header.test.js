import React from 'react';
import Header from './../Header';
import { render } from '@testing-library/react';

describe('Header', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
