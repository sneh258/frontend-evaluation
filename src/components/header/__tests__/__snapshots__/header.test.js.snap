// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="header_name"
    >
      EVENTIFY
    </div>
  </div>
</DocumentFragment>
`;
